convene-web/app/javascript/src/custom_components/button.scss

```diff
@@ -0,0 +1,9 @@
+.btn {
+  @apply font-bold py-2 px-4 rounded;
+}
+.btn-blue {
+  @apply bg-blue-500 text-white;
+}
+.btn-blue:hover {
+  @apply bg-blue-700;
+}
```

Comment:
Not blocking: I would prefer that we use the same file name as we use for the css classes. I.e. `button.scss` defineds the `.button` class.